src/containers/survey/SurveySagas.js

```diff
@@ -84,7 +87,10 @@ function* getChronicleUserAppsWorker(action :SequenceAction) :Generator<*, *, *>
       .toMap()
       .mapKeys((index, entity) => entity.getIn(['associationDetails', OPENLATTICE_ID_FQN, 0]))
       .map((entity, id) => entity.set('id', id))
-      .map((entity) => entity.setIn(['entityDetails', 'ol.title', 0], getAppNameFromUserAppsEntity(entity)));
+      .map((entity) => entity.setIn(['entityDetails', TITLE_FQN, 0], getAppNameFromUserAppsEntity(entity)))
+      .map((entity) => entity.setIn(['associationDetails', DATE_TIME_FQN],
+        [getMinimumDate(entity.getIn(['associationDetails', DATE_TIME_FQN]))]))
+      .sortBy((entity) => DateTime.fromISO(entity.getIn(['associationDetails', DATE_TIME_FQN, 0])));
```

Comment:
no change needed here, but it might be worth pulling `associationDetails` and `entityDetails` out into constants in another file.